invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/CreatibutorsField/CreatibutorsModal.js

```diff
@@ -355,7 +363,7 @@ export class CreatibutorsModal extends Component {
 
         this.updateIdentifiersAndAffiliations(
           formikProps,
-          personIdentifiers,
+          personIdentifiers.map((identifier) => identifier.identifier), // Only identifer value is needed for the form
```

Comment:
wasn't this done in line 344? I have a feeling it was more readable there... was it intentional to move it here?

Reply:
Yes, this is the main change. To not lose the `identifier.scheme` when setting the state.